app/app.js

```diff
@@ -45,20 +23,12 @@ module.exports = async (options) => {
 
   // Feature flags
   const flags = {
-    isDeployedToHeroku: !!HEROKU_APP
+    isDeployedToHeroku: !!process.env.HEROKU_APP
   }
 
-  // Share feature flags with middleware
-  env.addGlobal('flags', flags)
```

Comment:
We reference this in app/views/index.njk to avoid showing the link to the 'local' version of the Sass docs on the deployed review app:

https://github.com/alphagov/govuk-frontend/blob/56bd42d93ca653bfc6e62fac5dd5499f7e2aee10/app/views/index.njk#L45-L47

As `flags` is no longer a global in the Nunjucks environment, the link is now [showing on the preview app](https://govuk-frontend-pr-3224.herokuapp.com/).

Reply:
Well spotted, I've let Nunjucks grab it back from Express

```mjs
// Share feature flags with middleware
env.addGlobal('flags', app.get('flags'))
```

Reply:
Neat 🙌🏻